src/datasources/auth-api/siwe-api.service.ts

```diff
@@ -7,6 +7,8 @@ import { verifyMessage } from 'viem';
 
 @Injectable()
 export class SiweApi implements IAuthApi {
+  private readonly NONCE_LENGTH = 8;
```

Comment:
I'd set it as `static`.

Reply:
Changed in e607443959055f523163be550fa5670763fc96d7.